packages/alea-frontend/pages/api/get-section-info/[courseId].ts

```diff
@@ -9,7 +9,7 @@ import { CoverageSnap } from '@stex-react/utils';
 import { convert } from 'html-to-text';
 import { NextApiRequest, NextApiResponse } from 'next';
 import { getCoverageData } from '../get-coverage-timeline';
-import { readFileSync } from 'fs';
+import { existsSync, readFileSync } from 'fs';
 
 let processedSlidesJson: any = null;
```

Comment:
We need multiple entries for each course (Record<string, ...your obj...>)
{
 ai-1: ...
 lbs: ...
 }